pkg/router/router_test.go

```diff
@@ -114,6 +114,16 @@ func TestMain(m *testing.M) {
 		ReloadFn:              func(shutdown bool) error { return nil },
 		TemplatePath:          "../../images/router/haproxy/conf/haproxy-config.template",
 		ReloadInterval:        reloadInterval,
+		HTTPResponseHeaders: []templateplugin.HTTPHeader{{
+			Name:   "x-foo",
+			Value:  "'bar'",
+			Action: "Set",
+		}},
+		HTTPRequestHeaders: []templateplugin.HTTPHeader{{
+			Name:   "x-quoted",
+			Value:  `'"shouldn'\''t break"'`,
```

Comment:
The fact that the template config has to have the values already quoted seems wrong.  In my opinion, the template plugin should do the quoting.